packages/SwingSet/src/hasher.js

```diff
@@ -0,0 +1,32 @@
+import { assert } from '@agoric/assert';
+
+import { createHash } from 'crypto';
```

Comment:
Are we comfortable with the node `crypto` package?

Reply:
Yeah, it's the standard one, basically just a wrapper around OpenSSL, and their hashing functions are pretty straightforward. I compared the results against `/usr/bin/sha256sum`.